### PR TITLE
Add docs for Ruby buildpack Rails database configuration

### PR DESCRIPTION
**Pull request by RochesterinNYC**
- Remove vague section about Rails 4 and database config from ruby
  buildpack services docs

[#128506411]

Signed-off-by: Sam Smith <sesmith177@gmail.com>

### CODE REVIEW

**Comment by bentarnoff** [comment]
Thanks, @RochesterinNYC !

File: ruby/gsg-ror.html.md.erb [hunk 1]
```diff
@@ -47,13 +47,22 @@ You can specify app deployment options in a manifest that the `cf push` command
 
 <%=vars.product_short%> uses the default standard Ruby web server library, WEBrick, for Ruby and RoR apps. However, <%=vars.product_short%> can support a more robust production web server, such as Phusion Passenger, Puma, Thin, or Unicorn. If your app requires a more robust web server, refer to the [Configuring a Production Server](../prod-server.html) topic for help configuring a server other than WEBrick.
 
-## <a id="login"></a> Step 3: Log in and Target the API Endpoint ##
+## <a id="database-configuration"></a> Step 3: Create the Database Configuration ##
+
+For Rails versions 4.0.x and earlier, the Ruby buildpack will overwrite the `config/database.yml` file using the contents of the DATABASE_URL environment variable. This variable can be set by the user or will be automatically set
```
Comment (sclevine) [inline ruby/gsg-ror.html.md.erb]: Might be more clear to say `4.0.0 and earlier`

File: ruby/gsg-ror.html.md.erb [hunk 2]
```diff
@@ -47,13 +47,22 @@ You can specify app deployment options in a manifest that the `cf push` command
...
+based on [bound service instances](./ruby-service-bindings.html#vcap-services-defines-database-url).
+
+For Rails versions 4.1.y and later, the Ruby buildpack will not modify the `config/database.yml` file.
```
Comment (sclevine) [inline ruby/gsg-ror.html.md.erb]: Might be more clear to say `4.1.0 and later`

**Comment by bentarnoff** [comment]
Thanks, @sclevine, we'll make those changes.

**Pull request merged**